Backpack/Calendar/Classes/BPKCalendar.m

```diff
@@ -192,7 +193,9 @@ - (void)layoutSubviews {
         CGRectMake(width / 2.0 - yearPillWidth / 2.0, CGRectGetHeight(self.calendarWeekdayView.frame) + BPKSpacingLg,
                    yearPillWidth, yearPillHeight);
 
-    [self.calendarView.collectionViewLayout invalidateLayout];
+    if (calendarWidth != CGRectGetWidth(self.calendarView.frame)) {
```

Comment:
Given that the underlying type here is `CGFloat` which is a floatingpoint value do we wanna be doing direct comparisons?

See this [SO question](https://stackoverflow.com/questions/10334688/how-dangerous-is-it-to-compare-floating-point-values)

Reply:
I wanted to do that but completely forgot about it. Thanks, I'll fix it!